Polish up APISIX metrics to follow Service concept (#9775)
apache/skywalking: docs/en/setup/backend/backend-apisix-monitoring.md

2. Set up [OpenTelemetry Collector ](https://opentelemetry.io/docs/collector/getting-started/#docker). For details on Prometheus Receiver in OpenTelemetry Collector, refer to [here](../../../../test/e2e-v2/cases/apisix/otel-collector/otel-collector-config.yaml).
3. Config SkyWalking [OpenTelemetry receiver](opentelemetry-receiver.md).

### APISIX Monitoring
[APISIX prometheus plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) provide multiple dimensions metrics for APISIX server , node , route , etc. 

> Accordingly, Skywalking observes the status, payload, and latency of the APISIX server, which is cataloged as a `LAYER: APISIX` `Service` in the OAP. Meanwhile, the APISIX instance would be recognized as `LAYER: APISIX` `instance`, and the route rule , node would be recognized as `LAYER: APISIX` `endpoint`.

Reviewer: ```suggestion
Accordingly, Skywalking observes the status, payload, and latency of the APISIX server, which is cataloged as a `LAYER: APISIX` `Service` in the OAP. Meanwhile, the instances would be recognized as `LAYER: APISIX` `instance`s. The route rules and nodes would be recognized as `endpoint`s with `route/` and `node/` prefixes.
```

#### Specify Skywalking Service name  

> Skywalking extract OTEL Collector attribute `skywalking_service` as `Service` name .

Reviewer: ```suggestion
SkyWalking expects OTEL Collector attribute `skywalking_service` to be the `Service` name.
```

> Accordingly, Skywalking observes the status, payload, and latency of the APISIX server, which is cataloged as a `LAYER: APISIX` `Service` in the OAP. Meanwhile, the instances would be recognized as `LAYER: APISIX` `instance`s. The route rules and nodes would be recognized as `endpoint`s with `route/` and `upstream/` prefixes.

Reviewer 2: ```suggestion
Accordingly, SkyWalking observes the status, payload, and latency of the APISIX server, which is cataloged as a `LAYER: APISIX` `Service` in the OAP. Meanwhile, the instances would be recognized as `LAYER: APISIX` `instance`s. The route rules and nodes would be recognized as `endpoint`s with `route/` and `upstream/` prefixes.
```

> #### Specify Skywalking Service name  

Reviewer 2: ```suggestion
#### Specify SkyWalking Service name  
```

> You could specify `skywalking_service` attribute by `labels` keyword if using Prometheus receiver , as following :

Reviewer: ```suggestion
Make sure `skywalking_service` attribute exists through `static_configs` of OTEL Prometheus scape config.
```

```yaml
receivers:
  prometheus:
    config: 
     scrape_configs:
       - job_name: 'apisix-monitoring' 
         static_configs:
           - targets: ['apisix:9091']
             labels:
               skywalking_service: exmple_service_name  # Specify Skywalking Service name              
```

You also could leverage OTEL Collector processor to add `service` attribute , as following :

```yaml      
processors:
  resource/skywalking-service:
    attributes:
    - key: skywalking_service   
      value: exmple_service_name # Specify Skywalking Service name 
      action: insert                     
```

> Note , if you don't specify `service` attribute , Skywalking would use `APISIX` as default service name  

Reviewer: ```suggestion
Notice , if you don't specify `service` attribute, SkyWalking OAP would use `APISIX` as the default service name  
```

Author: `skywalking_service` attribute

Reviewer: This is the original version issue.
